ci-conda.Dockerfile

```
@@ -24,115 +24,122 @@ ENV SCCACHE_IDLE_TIMEOUT=32768
 ENV SCCACHE_S3_USE_SSL=true
 ENV SCCACHE_S3_NO_CREDENTIALS=false
 
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
 # Install system packages depending on the LINUX_VER
-RUN \
-    case "${LINUX_VER}" in \
-      "ubuntu"*) \
-        echo 'APT::Update::Error-Mode "any";' > /etc/apt/apt.conf.d/warnings-as-errors \
-        && apt-get update \
-        && apt-get upgrade -y \
-        && apt-get install -y --no-install-recommends \
-          file \
-          unzip \
-          wget \
-        && rm -rf "/var/lib/apt/lists/*"; \
-        ;; \
-      "centos"* | "rockylinux"*) \
-        yum -y update \
-        && yum -y install --setopt=install_weak_deps=False \
-          file \
-          unzip \
-          wget \
-          which \
-          yum-utils \
-        && yum clean all; \
-        ;; \
-      *) \
-        echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1; \
-        ;; \
-    esac
+RUN <<EOF
+case "${LINUX_VER}" in
+  "ubuntu"*) 
+    echo 'APT::Update::Error-Mode "any";' > /etc/apt/apt.conf.d/warnings-as-errors
+    apt-get update
+    apt-get upgrade -y
+    apt-get install -y --no-install-recommends \
+      file \
+      unzip \
+      wget
+    rm -rf "/var/lib/apt/lists/*"
+    ;;
+  "centos"* | "rockylinux"*) 
+    yum -y update
+    yum -y install --setopt=install_weak_deps=False \
+      file \
+      unzip \
+      wget \
+      which \
+      yum-utils
+    yum clean all
+    ;;
+  *) 
+    echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1
```

```suggestion
    echo "Unsupported LINUX_VER: ${LINUX_VER}"
    exit 1
```

Let's remove the `&&` since we're no longer using it anywhere else.